server.js

```diff
@@ -18,8 +19,66 @@ import env from "./config/env.generated.json";
 const defaultPort = 3000;
 const PORT = env.PORT || process.env.PORT || defaultPort;
 
+/**
+ * Some app security settings
+ */
+
+var securityHeaders = {
+  directives: {
+    defaultSrc: [
+      `\'self\'`
+    ],
+    scriptSrc: [
+      `\'self\'`,
+      `\'unsafe-inline\'`
+    ],
+    fontSrc: [
+      `\'self\'`,
+      `https://code.cdn.mozilla.net`
+    ],
+    styleSrc: [
+      `\'self\'`,
+      `\'unsafe-inline\'`,
+      `https://code.cdn.mozilla.net`
+    ],
+    imgSrc: [
+      `\'self\'`,
+      `\'unsafe-inline\'`,
+      `data:`,
+      `*`
+    ],
+    connectSrc: [
+      `\'self\'`,
+      env.PULSE_API || `https://network-pulse-api-staging.herokuapp.com/`,
+      `*`
+    ]
+  },
+  reportOnly: false,
+  browserSniff: false
+};
+
+app.use(helmet.contentSecurityPolicy(securityHeaders));
```

Comment:
Nice to see a CSP header, we should consider doing more work later to remove the unsafe directives we're adding (unsafe-inline, data:, "*")